CHANGES.txt

```diff
@@ -35,6 +35,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       OverrideEnvironment. Enable env.setdefault() method, add tests.
     - Raise an error if an option (not otherwise consumed) is used which
       looks like an abbreviation of one one added by AddOption. (#3653)
+    - Fix three issues in MergeFlags: signature/return did not match
+      documentation or existing usage; merging --param arguments did
+      not work (issue #3107); passing a dict to merge where the values
+      are strings failed (issue #2961).
```

Comment:
Add a note that it no longer returns the passed environment. Then should be good to merge.